web/concrete/src/Asset/JavascriptAsset.php

```diff
@@ -1,80 +1,95 @@
 <?php
 namespace Concrete\Core\Asset;
+
 use HtmlObject\Element;
 use Config;
 
-class JavascriptAsset extends Asset {
+class JavascriptAsset extends Asset
+{
+
+    protected $assetSupportsMinification = true;
+    protected $assetSupportsCombination = true;
 
-	protected $assetSupportsMinification = true;
-	protected $assetSupportsCombination = true;
+    protected $scriptType = 'text/javascript';
```

Comment:
because it is set in the output directly. I could do this, but I would have to alter quite slightly the __toString method and the whole JavascriptAsset class (to create the element on the constructor first for instance, and only use it in the __toString method).

Plus that would be a performance leak as this would be done all the time, and not only when the asset is outputted( most of the assets are not outputted, but still registered).